toolbackup/src/main/resources/awsConfig.conf

```diff
@@ -0,0 +1,2 @@
+AWS_ACCESS_KEY: AKIAIOSFODNN7EXAMPLE
```

Comment:
The AKIA... is the actual key my s3 ninja gave me as a mock aws access key should I rename AWS_ACCESS_KEY then? I removed awsConfig.conf from git commit now